app/controllers/api/v1/sessions_controller.rb

```diff
@@ -51,14 +56,23 @@ def get_organization_list
         organization_list = Organization.all.select(:id, :name).order(id: :asc)
       elsif @user.is_admin?
         organization_list = ["id": @user.organization_id, "name": @user.organization.name]
-      elsif @user.is_department_head? || @user.is_resolver?
+      elsif @user.is_department_head? #|| @user.is_resolver?
         organization_list = [{ "id": @user.organization.id,
                                "name": @user.organization.name, 
                                "department_id": @user.department.id, 
-                               "department_name": @user.department.name }]
+                               "department_name": @user.department.name }]                                                         
       else
-        organization_list = ["id": @user.organization_id, "name": @user.organization.name]
+        organization_list = ["id": @user.organization_id, "name": @user.organization.name]                                                                                                                                                                                                    
       end
     end
+
+    def fetch_google_oauth_data
+      uri = URI(GOOGLE_ID_TOKEN_VERIFICATION_URL)
+      google_oauth_params = { id_token: permitted_params[:token] }
+      uri.query = URI.encode_www_form(google_oauth_params)
+      response = Net::HTTP.get_response(uri)
+      return JSON.parse(                                                                                                                                                                                                                                                                  response.body) if response.is_a?(Net::HTTPSuccess)
+      nil
```

Comment:
Why is it required to return nil explicitly?

Reply:
No there is no any specific need to return nil this function will return nil if the condition won't be satisfied